app/models/program.rb

```diff
@@ -165,7 +165,7 @@ def languages_list
   # * +False+ -> If there is not any event for the given date
   def any_event_for_this_date?(date)
     parsed_date = DateTime.parse("#{date} 00:00").utc
-    EventSchedule.where(schedule: selected_schedule).where(start_time: parsed_date..(parsed_date + 1)).any?
+    EventSchedule.where(schedule: selected_schedule).where(start_time: parsed_date..(parsed_date + 1.day)).any?
```

Comment:
Why do we need this?

Reply:
We need this because we are getting : 
![screen shot 2017-12-16 at 3 24 01 am](https://user-images.githubusercontent.com/20709985/34062061-bbe485d0-e210-11e7-9ee9-10880b0eea73.png)

But the behaviour should be : 
![screen shot 2017-12-16 at 3 24 21 am](https://user-images.githubusercontent.com/20709985/34062067-c80285ec-e210-11e7-9e31-80693bdd9207.png)

Adding 1 to parsed_date just increase the parsed_date by 1 sec not by one day

Reply:
Basically, this is a bug.

Reply:
Why aren't you using just Time so that the rubocop change is not needed at all?
`Time.new(2017, 12, 21, 10, 00, 00, "+00:00")`

Also did you come across other occurrences of this issue?

Reply:
We can also use this but using DateTime is better when we need date with time instead of using hardcoded date with time.

Reply:
And i am also getting offence in this line. Either way we have to change the rubocop.

![screen shot 2017-12-21 at 12 46 12 am](https://user-images.githubusercontent.com/20709985/34224349-8fd994b8-e5e8-11e7-92ed-0f0e2553d8f2.png)